pkg/operator/ceph/cluster/mon/service.go

```diff
@@ -50,6 +51,11 @@ func (c *Cluster) createService(mon *monConfig) (string, error) {
 		s.Spec.ClusterIP = v1.ClusterIPNone
 	}
 
+	// If deploying Nautilus or newer we need a new port for the monitor service
+	if cephv1.VersionAtLeast(c.cephVersion.Name, cephv1.Nautilus) {
+		addServicePort(s, "msgr2", 3300)
```

Comment:
Existing mons won't get this port because there is no call to `Update()`. So existing mons are ok to just continue to use v1, right?

Reply:
Even if they get upgrades to Nautilus? During the upgrade they should have this port added.

Reply:
Currently it won't be added. We would need to call `Update()` after line 64 below. So the sequence might look like this:
```
Create(s)
if already exists (s) {
  Get()
  if nautilus port not set {
    Update(s)
  } 
}
```

Reply:
I still need to address this.